includes/class-accredible-learndash-admin-menu.php

```diff
@@ -136,7 +114,7 @@ public static function add_action_links( $links ) {
 			$mylinks = array(
 				'<a href="' . admin_url( 'admin.php?page=accredible_learndash_settings' ) . '">Settings</a>',
 			);
-			return array_merge( $links, $mylinks );
+			return array_merge( $mylinks, $links );
```

Comment:
To change the order of the links on the plugin page to "Settings, Deactivate".

![Screen Shot 2022-03-30 at 15 20 58](https://user-images.githubusercontent.com/20009662/160764723-a7246322-857b-4b76-81f7-d630b0b34d80.png)